<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmProofControlForm</class>
 <widget class="QWidget" name="FlyEmProofControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="topHorizontalLayout">
     <item>
      <widget class="QPushButton" name="dvidPushButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { color: green; }</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/open_dvid.png</normaloff>:/images/open_dvid.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="dataInfoWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Line" name="line_5">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font color=&quot;Green&quot;&gt;View&lt;/font&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="segmentationHorizontalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Segmentation Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="smallRadioButton">
       <property name="text">
        <string>Small</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bigRadioButton">
       <property name="text">
        <string>Big</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fullRadioButton">
       <property name="text">
        <string>Full</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="segmentSizePushButton">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>3D Body Visualization</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="bodyViewPushButton">
          <property name="toolTip">
           <string>3D body visualization (high resolution ball cloud)</string>
          </property>
          <property name="text">
           <string>(H) Ball Cloud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coarseBodyPushButton">
          <property name="toolTip">
           <string>3D body visualization (low resolution ball cloud)</string>
          </property>
          <property name="text">
           <string>(L) Ball Cloud</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="meshPushButton">
          <property name="toolTip">
           <string>3D body visualization (high resolution mesh)</string>
          </property>
          <property name="text">
           <string>(H) Mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coarseMeshPushButton">
          <property name="toolTip">
           <string>3D body visualization (low resolution mesh)</string>
          </property>
          <property name="text">
           <string>(L) Mesh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="skeletonViewPushButton">
        <property name="toolTip">
         <string>3D skeleton visualization</string>
        </property>
        <property name="text">
         <string>Skeleton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="Line" name="line_6">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#008000;&quot;&gt;Edit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Correct False Split</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select multiple bodies</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>  =&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mergeSegmentPushButton">
            <property name="toolTip">
             <string>Group selected bodies (M)</string>
            </property>
            <property name="text">
             <string>Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>=&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadPushButton">
            <property name="toolTip">
             <string>Upload merging result (U)</string>
            </property>
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Correct False Merge</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select a single body</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>  =&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="splitPushButton">
            <property name="toolTip">
             <string>Enter split mode</string>
            </property>
            <property name="text">
             <string>Start Split</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8"/>
   </item>
   <item>
    <widget class="ZFlyEmBookmarkWidget" name="bookmarkWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>dataInfoWidget</zorder>
  <zorder>bookmarkWidget</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmBookmarkWidget</class>
   <extends>QWidget</extends>
   <header location="global">flyem/widgets/zflyembookmarkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
